sootup.core/src/main/java/sootup/core/graph/DominanceFinder.java

```diff
@@ -76,7 +102,8 @@ public DominanceFinder(StmtGraph<?> blockGraph) {
           continue;
         }
         int blockIdx = blockToIdx.get(block);
-        List<BasicBlock<?>> preds = new ArrayList<>(block.getPredecessors());
+        List<?> predecessors = block.getPredecessors();
```

Comment:
is predecessors used somewhere?

Reply:
Sorry, it's my mistake, I'll remove that